OKTA-809892 Add API service integration in OIN Manager docs (#5279)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/submit-app/main/apiservice/submit.md

#### API Service Settings

The **Client Credentials** grant type is selected. This is the only supported grant type for OAuth 2.0 API service integrations.

* **Enable scopes**: Enter the Okta API scope to grant access from your integration. See [Scope selection](/docs/guides/build-api-integration/main/#scope-selection).

    Click **+ Add Another** to add more than three scopes.

* **Link to configuraton guide**: Specify the URL to the configuration document for your integration. See [Customer configuration document guidelines](/docs/guides/submit-app-prereq/main/#customer-configuration-document-guidelines).

### Test your integration

You can test your API service integration in the OIN Manager before you submit it. Generate an instance of your service app in the OIN Manager and make a few API requests to verify that your integration works as intended.

#### Generate an instance

Generate a service app instance in your associated Okta Developer Edition org for testing. The app instance that you generate provides credentials for your API calls:

1. From the **API Service** tab, click **Test in Okta** at the bottom of your page.

>    Your browser redirects to the Authorize integration page in your Okta org.

Reviewer: ```suggestion
   Your browser redirects to the **Authorize integration** page in your Okta org.
```

Author: ```suggestion
   Your browser redirects to the **Authorize {app}** page in your Okta org.
```

> 1. From the Authorize integration page, click **Install and Authorize**.

Reviewer: ```suggestion
1. On the **Authorize integration** page, click **Install and Authorize**.
```

Author: ```suggestion
1. From the **Authorize** page, click **Install and Authorize**.
```

1. Copy the client secret from the dialog and securely store it for your integration test.
1. Click **Done** to dismiss the dialog.

> 1. On the integration details page, copy the Okta domain and client ID and securely store them for your integration test.

Reviewer: ```suggestion
1. On the **Integration details** page, copy the Okta domain and client ID and securely store them for your integration test.
```

Author: ```suggestion
1. On the **General** tab, copy the Okta domain and client ID and securely store them for your integration test.
```
